### DISCUSSION TITLE
feature flag needed in next 15 for server actions

### DISCUSSION THREAD

**Discussion by arboledavargas** [vercel/next.js#73855] [Help]
### Summary

when I run my application I receive this error:

Error: 
  × To use Server Actions, please enable the feature flag in your Next.js config. Read more: https://nextjs.org/docs/app/building-your-application/data-fetching/forms-and-mutations#convention

this is my package.json

  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint",
    "generate:graphql": "graphql-codegen"
  },
  "dependencies": {
    "@apollo/client": "^3.11.0-rc.4",
    "@apollo/experimental-nextjs-app-support": "^0.11.7",
    "@auth/core": "^0.37.4",
    "@auth/nextjs": "^0.0.0-380f8d56",
    "@auth0/nextjs-auth0": "^4.0.0-beta.0",
    "@graphql-codegen/cli": "^5.0.3",
    "@graphql-codegen/client-preset": "^4.5.1",
    "next": "15.0.4",
    "next-auth": "^5.0.0-beta.25",
    "react": "^19.0.0",
    "react-dom": "^19.0.0"
  },
  "devDependencies": {
    "@types/node": "^20",
    "@types/react": "^19",
    "@types/react-dom": "^19",
    "typescript": "^5"
  }

**Comment by icyJoseph** [comment]
Got a link to a repository? This sounds like some mismanaged dependencies... the flag hasn't been needed for a while
